packages/common-library/src/common_library/json_serialization.py

```diff
@@ -62,6 +62,8 @@ def decimal_encoder(dec_value: Decimal) -> int | float:
 
 
 ENCODERS_BY_TYPE: dict[type[Any], Callable[[Any], Any]] = {
+    AnyHttpUrl: str,
+    AnyUrl: str,
```

Comment:
nice. nevertheless are these not inheriting from each other? just wondering if the code should not be changed so that such cases are easier to fix by just putting the base class. Nevertheless thanks!